packages/fs-detectors/src/monorepos/get-monorepo-default-settings.ts

```diff
@@ -22,6 +22,18 @@ export class MissingBuildTarget extends Error {
   }
 }
 
+function supportsRootCommand(turboSemVer: string | undefined) {
+  if (!turboSemVer) {
+    return false;
+  }
+
+  if (!semver.validRange(turboSemVer)) {
+    return false;
+  }
```

Comment:
@nathanhammond We decided to fix this such that any tags (which are invalid ranges) will NOT use the new command for now to resolve the issue.

We could say tags like `latest` and `canary` should use the new command, but to do that completely, we'd have to check the lockfile. Would you like to follow-up with that fix or leave this as-is?

Reply:
The `fs-detectors` setup is completely unaware of lockfiles and that would require significant rearchitecting to support N different versions of lockfiles for M different package managers, as well as possible non-existence.

This was originally opting people in for a conservative, known-safe setup and your change continues in that vein.

In general we want a world where people don't install `turbo` in their project—especially if they're just using `latest` or `canary`.